templates/CRM/Contribute/Form/Contribution.tpl

```diff
@@ -241,52 +241,58 @@
   </table>
 
   {if !$contributionMode}
-    <fieldset class="payment-details_group">
-      <legend>
-        {ts}Payment Details{/ts}
-      </legend>
-        <table class="form-layout-compressed" >
-          <tr class="crm-contribution-form-block-payment_instrument_id">
-            <td class="label">{$form.payment_instrument_id.label}</td>
-            <td {$valueStyle}>{$form.payment_instrument_id.html} {help id="payment_instrument_id"}</td>
-            </td>
-          </tr>
-          {if $showCheckNumber || !$isOnline}
-            <tr id="checkNumber" class="crm-contribution-form-block-check_number">
-              <td class="label">{$form.check_number.label}</td>
-              <td>{$form.check_number.html}</td>
-            </tr>
-          {/if}
-          <tr class="crm-contribution-form-block-trxn_id">
-            <td class="label">{$form.trxn_id.label}</td>
```

Comment:
ok - transaction field, but I do wonder with this & check_number if we should 'bubble up' payment edits to the contribution column. Possibly multiple - ie. if there were 3 checks we could simply put 123|567|890 into civicrm_contribution.check_number. This feels compatible with existing stuff to me.